pangeo/values.yaml

```diff
@@ -20,7 +20,7 @@ jupyterhub:
   singleuser:
     image:
       name: pangeo/notebook
-      tag: e02bc75c40df01454dd01bd6ae55e0213ada7563
+      tag: 2018-05-19.2
```

Comment:
chartpress should hypothetically set this tag automatically during the build process, but I haven't verified that that actually works.